.github/ISSUE_TEMPLATE/new_bug_report.yaml

```diff
@@ -0,0 +1,151 @@
+name: New Bug report
+about: Create a report to help us improve Amplify CLI
+title: ""
+issue_body: true
+body:
+- type: markdown
+  attributes:
+    value: Thanks for taking the time to fill out this bug report! Try to include as much information as you can. 
+- type: markdown
+  attributes:
+    value: For reference, the current version of the Amplify CLI is <img src="https://img.shields.io/npm/v/@aws-amplify/cli.svg" />.
+- type: markdown
+  attributes:
+    value: |
+      > **Note**: If your issue is regarding the AWS Amplify Console service, please log it in the [Amplify Console GitHub Issue Tracker](https://github.com/aws-amplify/amplify-console/issues).
+- type: checkboxes
+  attributes:
+    label: |
+      Before opening, please confirm.
+    options:
+      - label: I have installed the latest version of the Amplify CLI (see above), and confirmed that the issue still persists.
+        required: true
+      - label: I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-cli/issues?q=is%3Aissue+).
+        required: true
+      - label: I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#bug-reports).
+        required: true
+      - label: I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
+      - required: true
+- type: markdown
+  attributes:
+    value: |
+      ## Environment
+- type: input
+  attributes:
+    label: How did you install the Amplify CLI?
+    description: "For example: npm, yarn, etc."
+- type: input
+  attributes:
+    label: If applicable, what version of Node.js are you using?
+    description: You can use `node -v` to check the node version on your system.
+- type: input
+  attributes:
+    label: Amplify CLI Version
+    description: You can use `amplify -v` to check the amplify cli version on your system.
+    placeholder: "4.44.2"
```

Comment:
This placeholder is going to be out of date very quickly. Maybe have the description here be the placeholder.